custom_components/localtuya/light.py

```diff
@@ -36,6 +39,56 @@
 DEFAULT_LOWER_BRIGHTNESS = 29
 DEFAULT_UPPER_BRIGHTNESS = 1000
 
+WHITE_MODE = "white"
+COLOR_MODE = "colour"
+SCENE_MODE = "scene"
+MUSIC_MODE = "music"
+CUSTOM_SCENE = "Custom"
```

Comment:
Same here: `SCENE_CUSTOM` and for music below

Reply:
Done